frontend/e2e-tests/dom-to-db-tests/data-entry.e2e.ts

```diff
@@ -730,3 +731,77 @@ test.describe("navigation", () => {
     });
   });
 });
+
+test.describe("resume data entry flow", () => {
```

Comment:
@praseodym I should have thought about this yesterday, but these tests overlap with the ones in `abort-data-entry.e2e.ts`.

Reply:
to be fixed in issue #384